src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/CreateVideoBookingRequest.kt

```diff
@@ -124,14 +135,18 @@ data class Appointment(
   @Schema(description = "End time for the appointment on the day", example = "11:45")
   @JsonFormat(pattern = "HH:mm")
   val endTime: LocalTime?,
-
-  @field:Size(max = 120, message = "The video link should not exceed {max} characters")
-  @Schema(description = "The video link for the appointment. Must be a valid URL", example = "https://video.here.com")
-  val videoLinkUrl: String?,
 ) {
-  @AssertTrue(message = "The supplied video link for the appointment is not a valid URL")
-  private fun isInvalidUrl() = videoLinkUrl == null || runCatching { URI(videoLinkUrl!!).toURL() }.isSuccess
-
   @AssertTrue(message = "The end time must be after the start time for the appointment")
   private fun isInvalidTime() = (startTime == null || endTime == null) || startTime.isBefore(endTime)
 }
+
+enum class AppointmentType {
+  // Probation types
+  RECALL_REPORT,
+  PRE_SENTENCE_REPORT,
+
+  // Court types
+  PRE_CONFERENCE,
+  HEARING,
```

Comment:
I will mirror these descriptions in the reference_codes data too - was just going to expand the court types into PRE, MAIN and POST.